src/main/webapp/common.js

```diff
@@ -15,6 +15,7 @@
 /**
  * Common methods for both the app page.
  */
+var GoogleMapsApiKey= "";
```

Comment:
You can't really use a global variable for this, as you'll effectively need to fetch it before you can use it (and in general you shouldn't assume the fetch already succeeded by the time the user clicks the button).

Here's how Ana-Maria did it for another page: https://github.com/googleinterns/step235-2020/blob/master/src/main/webapp/bookScript.js#L10

But seriously, resist the urge of solving all the world's problems in a single PR. Just add a TODO to getAddressFromLatLng() to add the right apikey and figure out a way to plumb it in another PR.

Reply:
Also... please use `let` instead of `var`

Reply:
I added it as a TODO.